commonmark/src/main/java/org/commonmark/internal/InlineParserImpl.java

```diff
@@ -135,9 +135,23 @@ public static BitSet calculateSpecialCharacters(BitSet delimiterCharacters) {
     private static void addDelimiterProcessors(Iterable<DelimiterProcessor> delimiterProcessors, Map<Character, DelimiterProcessor> map) {
         for (DelimiterProcessor delimiterProcessor : delimiterProcessors) {
             char opening = delimiterProcessor.getOpeningCharacter();
-            addDelimiterProcessorForChar(opening, delimiterProcessor, map);
             char closing = delimiterProcessor.getClosingCharacter();
-            if (opening != closing) {
+            if (opening == closing) {
+                DelimiterProcessor old = map.get(opening);
+                if(old != null && old.getOpeningCharacter() == old.getClosingCharacter()) {
```

Comment:
Missing space after `if`. Would you mind reformatting this and the other added lines?